include/hax_interface.h

```diff
@@ -169,6 +174,7 @@ struct hax_module_version {
 #define HAX_CAP_64BIT_RAMBLOCK     (1 << 3)
 #define HAX_CAP_64BIT_SETRAM       (1 << 4)
 #define HAX_CAP_TUNNEL_PAGE        (1 << 5)
+#define HAX_CAP_GPA_PROTECTION     (1 << 6)
```

Comment:
OK. 
The name came from my original design thought about the API. At that time, I was struggling what was used for protection index, hva or gpa. Later I decided to use GPA, although QEMU needs to maintain slots information in this case. But this sounds right direction to go.

Reply:
We could actually return both the GPA and HVA to QEMU at every `HAX_EXIT_PAGEFAULT`, and also allow the new IOCTL to be called either with a GPA range or HVA range (e.g. distinguished by a bit in the `flags` parameter). That way, we wouldn't even need the `gpa2hva` or `hva2gpa` conversion functions in QEMU, but it would require some API changes to the current snapshot support code in Android Emulator...